### PR TITLE
IGAPP-1200: Fix navigation bar arrows

### PR DESCRIPTION
**Pull request by steffenkleinle**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Comment by steffenkleinle** [comment]
Fixes unresponsive behavior of arrows on first render and coloring of right arrow if scrolled to end.

File: native/src/components/NavigationTiles.tsx [hunk 1]
```diff
@@ -52,16 +53,15 @@ const NavigationTiles = ({ tiles, theme }: NavigationTilesProps): ReactElement =
     setPercentageScrolled(contentOffset.x / (contentSize.width - layoutMeasurement.width))
   }
 
+  const scrolledToStart = percentageScrolled === 0
+  const scrolledToEnd = percentageScrolled >= scrolledToEndThreshold
+
+  const scrollToStart = () => scrollViewRef.current?.scrollTo({ x: 0, y: 0, animated: true })
+  const scrollToEnd = () => scrollViewRef.current?.scrollToEnd({ animated: true })
```
Comment (LeandraH) [inline native/src/components/NavigationTiles.tsx:60]: I guess this is fine as long as we don't get more icons there. Which I think is not planned?

**Review by LeandraH — APPROVED** [review]
Looks good, tested on Android emulator

File: native/src/components/NavigationTiles.tsx (hunk 1 above)
Comment (steffenkleinle) [reply native/src/components/NavigationTiles.tsx:60]: Agreed and yes, don't think more icons are planned :+1:

**Review by f1sh1918 — COMMENTED** [review]
The related issue is fixed so far but when using horizontal scrolling by touch the arrows were not disabled if they are reaching the end or start

https://user-images.githubusercontent.com/37902063/214334140-8955466c-03bb-42e0-b84a-c007e6af5c05.mp4

**Comment by steffenkleinle** [comment]
> The related issue is fixed so far but when using horizontal scrolling by touch the arrows were not disabled if they are reaching the end or start, f.e. scrolling to "Map" should disabled right arrow
> Simulator.Screen.Recording.-.iPhone.SE.2nd.generation.-.2023-01-24.at.16.18.32.mp4

Can't reproduce, can you play around with the thresholds a bit? What is the percentage scrolled that is left?

**Review by f1sh1918 — APPROVED** [review]
Bug only happens on ios if the horizontal scrolling is strong. Dunno if we need a bug ticket for that

**Pull request merged**